### DISCUSSION TITLE
How to query multiple database tables for options in the same form?

### DISCUSSION THREAD

**Discussion by MrChadMWood** [Budibase/budibase#12178] [Help] [answered]
I have a form that allows users to select values from various source tables.
However, I can not bind anything to these as they are technically not in the same form. 

The end goal is to let users select values from source tables, and send that data as a form like normal. What would be the recommended approach for doing this?

Edit: Image for clarity
![image](https://github.com/Budibase/budibase/assets/90083658/1a80f3ba-7cbc-467a-8969-6826946fbd37)

**Comment by melohagan** [comment, marked as answer]
Hey @MrChadMWood 

You could always use an automation with an app action trigger. Then you could pass the various form fields into the automation to be saved to the various tables.

- https://docs.budibase.com/docs/app-action
- https://docs.budibase.com/docs/automation-steps
